FinnUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'FinnUI'
-  s.version      = '23.3.0'
+  s.version      = '23.3.1'
```

Comment:
Better to use the fastlane script for making a new version

Reply:
Ok, should have read the readme 😄 Next time will use the script then.